src/admin/dumb_components/custom_email_form.js

```diff
@@ -0,0 +1,33 @@
+const React = require('react')
+const { map, objOf, zipWith, merge, compose } = require('ramda')
+
+export default class CustomEmail extends React.Component {
```

Comment:
Also we use `React.createClass` in this project so far, so may as well stick with that when we need it